integration_test/helper.sh

```diff
@@ -111,6 +113,12 @@ function wait_for_cmd() {
     return 1
 }
 
+function get_amoc_logs() {
```

Comment:
I don't see this function used anywhere, am I missing something?

Reply:
I used it for troubleshooting issue at CI environment and I think it's good to have in helpers.sh just in case.